src/testParser.ts

```diff
@@ -375,7 +381,7 @@ async function parseSuite(
       // fish the time-taken out of the test case attributes, if present
       const testTime = testcase._attributes.time === undefined ? '' : ` (${testcase._attributes.time}s)`
 
-      core.info(`${resolvedPath}:${pos.line} | ${message.replace(/\n/g, ' ')}${testTime}`)
+      core.info(`${resolvedPath}:${pos.line} | ${message.split('\n', 1)[0]}${testTime}`)
```

Comment:
In the event that `message` is a full stack trace, it seems more polite to just log the first line. This should be the only change that impacts existing users of the action.